<?xml version="1.0"?>
<!-- vim: set fenc=utf-8 et sts=4 ft=xml : -->
<launch>
	<node
		pkg="gpuimageproc"
		type="gpuimageproc_node"
		name="gpuimageproc"
		output="screen"
		clear_params="true"
		launch-prefix2="gdb -ex run --args"
	>
		<remap to="/loitor_stereo_visensor/left/image_raw" from="left/image_raw"/>
		<remap to="/loitor_stereo_visensor/right/image_raw" from="right/image_raw"/>
		<remap to="/loitor_stereo_visensor/left/camera_info" from="left/camera_info"/>
		<remap to="/loitor_stereo_visensor/right/camera_info" from="right/camera_info"/>

		<param name="approximate_sync" type="bool" value="false"/>
		<!-- param name="camera_info_file_left" type="str" value="/data/git/temp_ws/src/gpuimageproc/test/stereobm/euroc_left_scale_0.yaml"/ -->
		<!-- param name="camera_info_file_right" type="str" value="/data/git/temp_ws/src/gpuimageproc/test/stereobm/euroc_right_scale_0.yaml"/ -->
	</node>
</launch>
